lib/utils.js

```diff
@@ -26,31 +24,37 @@ function populateCache (url) {
       return Winston.error('Non-200 status code: %d', res.statusCode)
     }
 
-    const title = res.body.title
-    const headers = res.body.headers
-
-    const batch = Cache.batch()
-
-    // Add each facility to the cache
-    res.body.rows.forEach(function (row) {
-      const key = row[0]
-      if (!key) {
-        return Winston.error('Facility missing value', {facility: row})
-      }
-      const body = Mustache.render(template, {
-        title: title,
-        headers: JSON.stringify(headers),
-        row: JSON.stringify(row),
-        width: headers.length
+    const levelKey = url.split('sqlViews/')[1] // levelKey = <UID>.json
+    Cache.deleteSubLevelCacheDB(levelKey, () => { // clear the existing cache before add new data
```

Comment:
This could cause empty results to be returned if requests come in between when the cache is cleared and when the new entries are added. We need to be smart about this.